dissect/target/helpers/fsutil.py

```diff
@@ -963,3 +965,23 @@ def resolve_link(
         entry = resolve_link(fs, entry, previous_links)
 
     return entry
+
+
+def decompress_and_read(path: TargetPath) -> bytes:
```

Comment:
Same comment here, if you can make the same changes here as in #131 I'll make sure that everything is merged in the correct order.